### DISCUSSION TITLE
115网盘建议添加Cookie导入功能

### DISCUSSION THREAD

**Discussion by cuijun8200** [jxxghp/MoviePilot#2611] [建议]
115网盘只允许一个设备扫码，另一个设备扫描后前面的自动强制退出，建议添加Cookie导入功能

**Comment by LONGATLer** [comment]
可以抓包微信小程序的cookie使用，这个cookie持续时间很长，而且不会因为其他设备登录失效

**Reply by cuijun8200** [reply]
非常感谢，问题已得到解决
